agent/agents/mysql/perfschema/perfschema.go

```diff
@@ -85,6 +85,7 @@ type PerfSchema struct {
 	q                    *reform.Querier
 	dbCloser             io.Closer
 	agentID              string
+	queryLength          int
```

Comment:
Same question as for PG

Reply:
Renamed to maxQueryLength on all places (will do same for postgres PR), because its better naming.